sos/plugins/block.py

```diff
@@ -51,4 +51,15 @@ def setup(self):
                     "fdisk -l %s" % disk_path
                 ])
 
+        lsblk_result = self.call_ext_prog("lsblk -P -o +FSTYPE -p")
```

Comment:
Perhaps we should also add `lsblk -f``... I don't think we're capturing FSTYPE anywhere.

Reply:
(we could also do it here - the code isn't really using `-P` name=value parsing at all, so a simple `in` test of the line would work fine. Then we can store the result in the report for analysis use as well).